validator/src/main/java/com/google/daq/mqtt/registrar/Registrar.java

```diff
@@ -107,6 +111,10 @@ public class Registrar {
   public static void main(String[] args) {
     ArrayList<String> argList = new ArrayList<>(List.of(args));
     Registrar registrar = new Registrar();
+    executeWithRegistrar(registrar, argList);
+  }
+
+  public static void executeWithRegistrar(Registrar registrar, ArrayList<String> argList) {
```

Comment:
does this need to be static?  Can't it just be a regular class method on Registrar?

Reply:
I moved this from main() which was static and was looking to be consistent but make the most minimal change. WDYT?

Reply:
New attempt, PTAL